cookbooks/symfony_no_flex.rst

```diff
@@ -0,0 +1,300 @@
+SymfonyCloud without Flex
+=========================
+
+SymfonyCloud experience is optimal when Symfony applications are using Flex, but
```

Comment:
The SymfonyCloud experience is optimized for Symfony applications using Flex, but ...